common/schema.h

```diff
@@ -469,6 +472,11 @@ namespace swss {
 
 #define STATE_FLOW_COUNTER_CAPABILITY_TABLE_NAME    "FLOW_COUNTER_CAPABILITY_TABLE"
 
+#define STATE_EEPROM_INFO_TABLE_NAME        "EEPROM_INFO"
+#define STATE_FAN_INFO_TABLE_NAME           "FAN_INFO"
+#define STATE_TEMPERATURE_INFO_TABLE_NAME   "TEMPERATURE_INFO"
+#define STATE_PSU_INFO_TABLE_NAME           "PSU_INFO"
```

Comment:
These lines are not related to log message config

Reply:
Yes, it doesn't.
That is why I put it in a separate commit.